Jenkinsfile

```diff
@@ -0,0 +1,149 @@
+library 'magic-butler-catalogue'
```

Comment:
looks like we can use Jenkinsfile for our public repos? For logger-node it looks like we use circleci- how come we're using Jenkins here? https://github.com/logdna/logger-node

Reply:
The Jenkins flow needed some significant changes before it could publish to the main registry (and not our private @answerbook one).  Those changes are done now, but didn't exist when the node client was written.

We can circle back now and add Jenkins to the node client.  I'll open a card.

Reply:
Card opened: https://logdna.atlassian.net/browse/LOG-7457